README.md

```diff
@@ -1,70 +1,162 @@
+
+_Safe Paths_ is a built on [React Native](https://reactnative.dev/docs/getting-started) v0.61.5
+
+## Contributing
+
+Read the [contribution guidelines](CONTRIBUTING.md).
 
 ## Architecture
 
-Please refer to `docs/Private Kit Diagram.png` for a basic overview on the sequencing of generalized events and services that are used by Safe Paths.
+View the [architecture diagram](docs/Private_Kit_Diagram.png) for a basic overview on the sequencing of generalized events and services that are used by Safe Paths.
 
 ## Developer Setup
 
-Run the ```dev_setup.sh``` (Linux/MacOS) or ```dev_setup.bat``` (Windows) for needed tools.  It is assumed that Android Studio and/or xcode (on macOS) is set up and configured correctly to run applications in the simulator.  For Android Studio, the dev_setup script can help you, but you still may need to configure at least one android virtual device (avd) from within Android Studio first before you can run the app.
+First, run the appropriate setup script for your system. This will install relevant packages, walk through Android Studio configuration, etc.
 
-## Running
+**Note:** You will still need to [configure an Android Virtual Device (AVD)](https://developer.android.com/studio/run/managing-avds#createavd) after running the script.
 
-```yarn``` must be installed for this project, as it does a better job installing dependendies across platforms.  The dev_setup script above should have done that for you.
+#### Linux/MacOS
 
-### Install modules to the correct locations
+```
+dev_setup.sh
+```
+
+#### Windows
+
+```
+dev_setup.bat
+```
 
-1. ```yarn install``` (always do this after a new clone or checkout)
-2. ```yarn install:pod``` (additional step for macOS if you want to do an iOS build)
+## Running
+
+**Note:** In some cases, these procedures can lead to the error `Failed to load bundle - Could not connect to development server`. In these cases, kill all other react-native processes and try it again.
 
-### Run the app in a simulator
+#### Android (Windows, Linux, macOS)
 
-To run in the Android simulator (Windows, Linux, macOS):
 ```
 npx react-native run-android
 ```
-or (on macOS only)
+
+Device storage can be cleared by long-pressing on the app icon in the simulator, clicking "App info", then "Storage", and lastly, "Clear Storage".
+
+#### iOS (macOS only)
+
 ```
+yarn install:pod ## only needs to be ran once
 npx react-native run-ios
 ```
 
-NOTE: In some cases, these procedures can lead to the error `Failed to load bundle - Could not connect to development server`. In these cases, kill all other react-native processes and try it again.
+Device storage can be cleared by clicking "Hardware" on the system toolbar, and then "Erase all content and settings".
 
-## Contributing
+### Release Builds
 
-Read the [contribution guidelines](CONTRIBUTING.md).
+Generating a release build is an optional step in the development process.
```

Comment:
And requires an 🍎 developer id to be provisioned (currently) though soon that will not be the case!

Reply:
So following those steps for Android, I was able to build with a self signer cert.

For the developer ID stuff on iOS, is that going to be soon enough to make it not worth mentioning?

Reply:
"release build" vs "development build" is kind of misleading in this context, I guess.

There are really two different environments to manage as a RN developer - Javascript release and native release. Basically, my thought is we should clarify the difference - "release" sounds like it's ready for the app store, when it's really an unsigned artifact talking to a metro bundler. That being said, it's not worth splitting hairs here. We could eventually add some links in to android and apple docs on the different types of builds available :)